Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -664,12 +664,12 @@ internal struct TaskCancelEvent {}
 
 internal class TaskHandler<Delegate: HTTPClientResponseDelegate>: RemovableChannelHandler {
     enum State {
-        case idle
-        case bodySent
-        case sent
-        case head
+        case inactive
```

Comment:
`idle` is probably better than `inactive` because the channel is actually `isActive = true`, right?

Reply:
done